docs/src/documentation/03-components/05-input/select/03-accessibility.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/select/accessibility/
+---
+
+## Accessibility
+
+The Select component has been designed with accessibility in mind.
+
+It supports keyboard navigation and includes the following properties that provide additional information to screen readers:
+
+| Name           | Type     | Description                                                                                                                                                                                                                                                                 |
+| :------------- | :------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ariaLabel      | `string` | Allows you to specify an `aria-label` attribute of the component.                                                                                                                                                                                                           |
+| ariaLabelledby | `string` | Allows you to specify an `aria-labelledby` attribute of the component. This attribute references the id(s) of element(s) that label(s) the Select, separated by a space. The elements with those ids can be hidden, so that their text is only announced by screen readers. |
```

Comment:
On the Seat component, for example, the part where we say the ids are separated by space, etc… are on a separate paragraph, so the table isn't too cluttered. But non-blocking, up to you.

Reply:
True, it looks better. Moved outside the table.